### DISCUSSION TITLE
Audio Glitches in Android?

### DISCUSSION THREAD

**Discussion by timelessnesses** [ppy/osu#22705] [Q&A]
Clip: https://youtu.be/w-M7kQId-3I. 
So I am playing osu!Lazer and after watching auto play a map with storyboard and video (it's very heavy I barely getting 30 fps) my audio got cutted out (at the very beginning of the video). I tried change audio source but this didn't help (you can see my screen recorder record audio being glitched)
I need to restart osu!lazer to actually make the audio works again.
Also seems like changing audio source anything other than default and change back to default reproduce this glitch.
I can't provide log files since I restarted game so many times the log likely wiped out.

**Comment by timelessnesses** [comment]
I will try reproduce and provide log to you guys

**Comment by peppy** [comment]
At the point the game is running at 30fps, I don't think we should expect audio to sound correct. Please wait for upcoming renderer and optimisations which will hopefully bring the visuals (and audio) up to a better standard.

**Reply by timelessnesses** [reply]
I got the [log](https://cdn.timelessnesses.me/r/log.zip)

**Reply by timelessnesses** [reply]
the log here came from me changing audio outputs back and forth
